volto/codewalkthrough.rst

```diff
@@ -0,0 +1,59 @@
+.. _code_walkthrough-label:
+
+================
+Code Walkthrough
+================
+
+Volto is based on React, Redux and React-Router. All the code is located in the
+:file:`src` folder. Inside the :file:`src` folder we used the default Redux
+folder structure.
+
+Actions
+=======
+
+Actions contains all the redux actions for fetching all backend data like;
+content, users etc.
+
```

Comment:
Please follow our word style-guide: https://docs.plone.org/about/contributing/word_choice.html

Reply:
Is "Actions" a proper name or file name? It's self-referential and doesn't make sense to me, IOW "Flubbers contains all the redux flubbers". Perhaps a Glossary term would improve its meaning?

If "Actions" is not a proper noun, then English grammar rules dictate that "Actions" is singular and the verb "contains" is singular. Else "Actions" is plural and the verb should be plural "contain".

Remove semi-colon `;`.